### ISSUE TITLE
Fast Tracker Prerequisites

### ISSUE THREAD

**Issue by istabosz** [openlegend/core-rules#423]
Is Fast Tracker meant to require Agility 5 and Perception 5?

On the website it's rendering as OR, instead of AND.

**Comment by ucffool** [comment]
Per the YML it's `AND` and has been for a while. Just clarifying it as a rendering issue for olrpg.com

**Comment by brianfeister** [comment]
Yup, should be AND, will leave this open for me to fix the website

**Comment by brianfeister** [comment]
This has been fixed.

**Comment by brianfeister** [comment]
Re-opening... I guess not on the website, but I'm not working on the website now. Shifted to the PDF files full time

**Comment by BuildAndDie** [comment]
So, about Fast Tracker... It mentions:

> You can move at your full speed while tracking your quarry rather than the reduced speed typically incurred.

But this reduced speed thing isn't mentioned anywhere else in the rules? I mean, I suppose to an extent it is a rule of common sense, that you would be slower tracking and doesn't need to be addressed directly in Core Rules? Especially as how much you are reduced could depend a lot on setting and campaign world, but thought I'd throw that out as a question.

**Comment by brianfeister** [comment]
I had noticed this recently and forgot to make a note of it. Thanks @BuildAndDie .

@istabosz thoughts on resolution, I think he's right... we don't explicitly state this elsewhere. If we want to remain vague intentionally we could say

> You can move unhindered while tracking your quarry, whereas tracking would typically slow you down.

**Comment by BuildAndDie** [comment]
I like keeping it vague, as I explained it to people in discord, the setting, campaign, and specific situation could vary so much that you wouldn't make specific rules for tracking in the core rules.

A soft mention like this in the feats is enough. Mostly it should be common sense.

> If you are tracking through a dense bog, your speed while tracking might be 1/4, but if you are tracking in the snow with no wind, then you  might have no reduction of speed b/c of ease to follow the tracks. If there is a reduction, it would be determined by the GM

**Comment by oconnor0** [comment]
My confusion around Fast Tracker is that "rather than the reduced speed
typically incurred" sounds like it's referencing a different rule when it
isn't. I'm fine with vague. Perhaps something like "The GM determines if
and when tracking would reduce your speed." Or Great Moustache's suggestion.
